app/views/ProfileView/index.js

```diff
@@ -361,6 +378,73 @@ class ProfileView extends React.Component {
 		);
 	}
 
+	renderNicknameField = () => {
+		const {
+			nickname
+		} = this.state;
+		const {
+			theme, serverVersion
+		} = this.props;
+
+		// Nickname was added to release 3.5.0
+		if (semver.lte(serverVersion, '3.5.0')) {
+			return null;
+		}
+
+		return (
+			<RCTextInput
+				editable
+				inputStyle={[
+					!true && styles.disabled
+				]}
+				maxLength={120}
```

Comment:
Where did you get this from?

Reply:
https://github.com/RocketChat/Rocket.Chat/blob/10ea922d89b05f358c656996913d6492edd4c417/server/methods/saveUserProfile.js#L58

Reply:
Should I put this link as a comment?

Reply:
😱  That's so ugly!
Yeah, please add the line linking to the server repo.
Thanks.

Reply:
Ok, Thanks!